### PR TITLE
Milestone 1: Project setup

### PR DESCRIPTION
**Pull request by tmampa**
# Bookstore (Bookly 📚)
## In this milestone:

- [ ] initialised React app.
- [ ] Added React Redux.
- [ ] Applied _feature folder_ structure.
- [ ] Added react router.
- [ ] Added reusable components

## Changes made 🆕:
- [ ] Created Book and CreateNewBook as reusable components.
- [ ] Nav only has 2 paths with a root page.
- [ ] Book component takes data as props and return it, also has a button to remove book.
- [ ] CreateNewBook component has a form for ttile and author and a submit button.
 _requested by_ @Meltrust

### CODE REVIEW

**Review by Meltrust — COMMENTED** [review]
Hi @tmampa, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that lacks a proper readme and no linters setup.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by tmampa** [comment]
> Hi @tmampa,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that lacks a proper readme and no linters setup.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Navbar />
+      <Switch>
+        <Route path="/books">
```
Comment (Meltrust) [inline src/App.js]: - [ ] Kindly please make this path your root path.  Look:

image

That way, on load your app will land always on the `<Books />` component. 👍

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const Navbar = () => (
+  <ul>
+    <li>
+      <Link to="/">Bookly</Link>
+    </li>
+    <li>
+      <Link to="/books">Books</Link>
+    </li>
+    <li>
+      <Link to="/categories">Categories</Link>
```
Comment (Meltrust) [inline src/components/Navbar.js]: - [ ] You only need to links on your nav.  Look:

image

The `/books` link won't be needed anymore.  

Look:

image

It still works!

That way, you will end up with the 2 required links on your nav, a link to `/` which is the booklist "bookly", and another one for the "categories" page, as stated on the requirements.

File: src/components/Books.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  <div>
+    <h3>Book list</h3>
+    <button type="button">Remove book</button>
+    <div>
+      <input type="text" placeholder="Add a book" />
+    </div>
+  </div>
+);
+
+export default Books;
```
Comment (Meltrust) [inline src/components/Books.js]: - [ ] We need to set up reusable components:

image

In order to comply with this requirement, the Books component should call other components inside it and render them.

The components that should be called are:

1.  The `Book` component (singular) renders each book:

image

2.  The `CreateNewBook` component, which is essentially a form:

image

Kindly please create these components as per the instructions above and include them inside the `Books` component.

Here is an example of how can those be included inside your `Books` component:

image

Note: The form inside your `CreateNewBook` component should have a field for the name of the book, and another for the author!

That way, your components will be reusable, and ready for the data! 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @tmampa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Nice readme.  Below I wrote some comments for you, please kindly go check them 👇 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @tmampa,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**